docs/sources/alerting/configure-notifications/manage-contact-points/_index.md

```diff
@@ -64,32 +79,33 @@ Complete the following steps to add a contact point.
 1. On the **Contact Points** tab, click **+ Add contact point**.
 1. Enter a descriptive name for the contact point.
 1. From **Integration**, select a type and fill out mandatory fields. For example, if you choose email, enter the email addresses. Or if you choose Slack, enter the Slack channel and users who should be contacted.
-1. Some contact point integrations, like email or webhook, have optional settings. In **Optional settings**, specify additional settings for the selected contact point integration.
+1. Some contact point integrations, like email or Webhook, have optional settings. In **Optional settings**, specify additional settings for the selected contact point integration.
 1. In Notification settings, optionally select **Disable resolved message** if you do not want to be notified when an alert resolves.
 1. To add another contact point integration, click **Add contact point integration** and repeat steps 6 through 8.
 1. Save your changes.
 
-## Edit a contact point
+## Use notification templates
 
-Complete the following steps to edit a contact point.
+Use templates in contact points to customize your notifications.
 
-1. In the left-side menu, click **Alerts & IRM** and then **Alerting**.
-1. Click **Contact points** to view a list of existing contact points.
-1. On the **Contact Points** tab, find the contact point you want to edit, and then click **Edit**.
-1. Update the contact point and save your changes.
+Complete the following steps to add templates to your contact point.
 
-## Delete a contact point
+1. Click an existing contact point or create a new one
+1. In **Optional settings**, click any field that contains templates.
 
-Complete the following steps to delete a contact point.
+   For example, if you are creating an email contact point integration, click **Message** or **Subject**.
 
-1. In the left-side menu, click **Alerts & IRM** and then **Alerting**.
-1. Click **Contact points** to view a list of existing contact points.
-1. On the **Contact Points** tab, find the contact point you want to delete, and then click **More** -> **Delete**.
-1. In the confirmation dialog box, click **Yes, delete**.
+1. Click **Edit**.
+   A dialog box opens where you can select templates.
+1. Click **Select existing template** to select a template and preview it using the default payload.
```

Comment:
Maybe we can add here that at this point you can click save, and that means using this single template in the field ?